SparkPost/support-docs
# WE-352: Migrate Automatic Inline Seeds User Guide

**What Changed?**

* Added `/docs/user-guide/automatic-inline-seeds`
* Added redirect from `/docs/user-guide/automatic-inline-seeding-user-guide` to this page (the AIS announcement email used this path)
* [Extra Credit] Updated `/docs/deliverability/deliverability-analytics` with most recent content update https://github.com/SparkPost/support-docs/commit/23042db7bcb6f119dcfa25f0db4ed75d72492baa

**How to Test?**

* Open `/docs/user-guide/automatic-inline-seeds` and confirm the page is rendering correctly with images
* Open `/docs/user-guide/automatic-inline-seeding-user-guide` and confirm you are successfully redirect to the page
* Confirm `/docs/deliverability/deliverability-analytics` has that attention grabbing announcement at the top
